### PR TITLE
[VSLCV] Use series option label and hardcoded string as fallback

### PR DESCRIPTION
**Pull request by srallen**
_Please request review from `@zooniverse/frontend` team. If PR is related to design, please request review from `@beckyrother` in addition._ 

Package: lib-classifier

Describe your changes:
I accidentally hardcoded in a label for the phase focus, so this updates the controls to make sure they use the series' options labels and only use the hardcoded values as fallback.

You can see the issue here:
Screen Shot 2020-09-02 at 4 17 36 PM

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/components/Controls/components/PhaseFocusControls/PhaseFocusControls.js [hunk 1]
```diff
@@ -23,9 +23,9 @@ function PhaseFocusControls(props) {
   return (
     <Box direction='column'  pad='none'>
       {data.map((series, seriesIndex) => {
-        const checked = phaseFocusedSeries === seriesIndex
-        const label = counterpart('VariableStarViewer.label', { id: seriesIndex + 1 })
         const { seriesOptions } = series
+        const checked = phaseFocusedSeries === seriesIndex
+        const label = seriesOptions.label || counterpart('VariableStarViewer.label', { id: seriesIndex + 1 })
```
Comment (shaunanoordin) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/components/Controls/components/PhaseFocusControls/PhaseFocusControls.js:28]: Do you need to check for `seriesOption`'s existence here? I noticed you checked for `series.seriesOptions?.label` in the container.

**Review by shaunanoordin — COMMENTED** [review]
## PR Review

Package: `lib-classifier`

This PR ensures that the labels on the Variable Star Light Curve Viewer (Subject Viewer subtype) follow the data-chart's pre-configured label values (if any), and only falls back to hardcoded label values when the config is absent.

Code looks good - I don't see any major issues with how the fallbacks work - but is there a VSLCV project I can run a user test on?

**Comment by shaunanoordin** [comment]
Ah, found it: `https://local.zooniverse.org:8080/?env=production&project=4878&workflow=16135` aka `adamamiller/stellar-sleuths`

**Review by shaunanoordin — APPROVED** [review]
## PR Review (Update)

Code looks good (sans minor query about an existence check) and manual testing looks good on Stellar Sleuths. LGTM. 👍

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/components/Controls/components/PhaseFocusControls/PhaseFocusControls.js (hunk 1 above)
Comment (srallen) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/components/Controls/components/PhaseFocusControls/PhaseFocusControls.js:28]:
```suggestion
        const label = seriesOptions?.label || counterpart('VariableStarViewer.label', { id: seriesIndex + 1 })
```

**Pull request merged**